### PR TITLE
strong name signing changes

### PR DESCRIPTION
**Pull request by rajan-chari**
Changes to allow strong name signing from a trusted build source.

### CODE REVIEW

File: bindings/cs/rl.net/rl.net.csproj [hunk 1]
```diff
@@ -26,4 +28,37 @@
...
+      <LastGenOutput>InternalsVisibleToTest.cs</LastGenOutput>
+    </None>
+    <Compile Include="InternalsVisibleToTest.cs" Exclude="@(Compile)">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>InternalsVisibleToTest.tt</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <Target Name="OutputVars" BeforeTargets="Build">
```
Comment (lokitoth) [inline bindings/cs/rl.net/rl.net.csproj]: Are these output vars still needed?

File: appveyor.yml [hunk 2]
```diff
@@ -22,3 +22,6 @@ build_script:
   - .\build-windows.cmd
 test_script:
   - .scripts\test.cmd
+# Debug appveyor builds
```
Comment (lokitoth) [inline appveyor.yml]: Is this still needed?

File: bindings/cs/rl.net/rl.net.csproj (hunk 1 above)
Comment (rajan-chari) [reply bindings/cs/rl.net/rl.net.csproj]: Still have a number of steps to finish.  I'd like to merge this to release with this on.  Can get rid of if eventually.

File: appveyor.yml (hunk 2 above)
Comment (rajan-chari) [reply appveyor.yml]: removed.

**Pull request merged**